build/package/helm/gateway/files/policies/policies.rego

```diff
@@ -13,7 +13,11 @@ allowed_paths := [
 	"/domain.ClusterAccess/GetClusterAccess",
 ]
 
-scoped_paths := [{"path": "/scim/", "scope": "WRITE_SCIM"}]
+scoped_paths := [{"scope": "WRITE_SCIM", "paths": [
```

Comment:
Additionally to the bug some paths which must be allowed to the API token with WRITE_SCIM where missing in the policy.